UnicornLeap/main.swift

```diff
@@ -41,8 +41,8 @@ func printImageError(_ filename: String) {
 }
 
 func printVerboseOutput() {
-  print("Seconds: \(command.seconds)")
-  print("Number: \(command.number)")
+  print("Seconds: \(String(describing: command.seconds))")
+  print("Number: \(String(describing: command.number))")
```

Comment:
Doesn't the string interp essentially do this for us? I guess I assumed that `command.number` was converted to a string implicitly here, but maybe I'm missing something!

Reply:
As it is an `Optional` it requires that to silence the warning 😄